plugins/tiddlywiki/tiddlyweb/tiddlywebadaptor.js

```diff
@@ -13,16 +13,23 @@ A sync adaptor module for synchronising with TiddlyWeb compatible servers
 "use strict";
 
 var CONFIG_HOST_TIDDLER = "$:/config/tiddlyweb/host",
-	DEFAULT_HOST_TIDDLER = "$protocol$//$host$/";
+	DEFAULT_HOST_TIDDLER = "$protocol$//$host$/",
+    	WikiExport = require('$:/core/modules/wiki.js');
```

Comment:
Why is the variable named "WikiExport"  ... There is no "export" handling going on. right?

Reply:
The wiki.js file exports a bunch of functions, and we use three of them. So I just called it that.

Reply:
I just had a look again.. IMO ``WikiExport` isn't used?!

Reply:
You're right. I missed that when I updated the other pages per our discussion here.